fbw-a380x/src/wasm/systems/a380_systems/src/fuel/fuel_quantity_management_system.rs

```diff
@@ -369,20 +364,21 @@ impl RefuelApplication {
         let zero_fuel_weight_cg_rounded = (zero_fuel_weight_cg_percent_mac).floor() as u32;
 
         let mut trim_fuel = Mass::default();
+        let target_zfw = self.trim_tank_map.params.target_zfw.clone();
+        let trim_tank_targets = self.trim_tank_map.trim_tank_targets.tables.clone();
```

Comment:
```suggestion
        let target_zfw = &self.trim_tank_map.params.target_zfw;
        let trim_tank_targets = &self.trim_tank_map.trim_tank_targets.tables;
```
No need to clone if you also do the other suggestions.